spec/spec_helper.rb

```diff
@@ -1,6 +1,6 @@
 require 'capybara/rspec'
 require 'rails_helper'
-require 'support/factory_girl'
+require 'factory_girl_rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.